### ISSUE TITLE
about loss of dis and gen

### ISSUE THREAD

**Issue by day2nightya** [edward3862/LoFGAN-pytorch#13]
![image](https://user-images.githubusercontent.com/105911615/175022267-4b7aa164-7914-42ba-a5c6-47d0898b9a0d.png)
The left is generator'loss, and the right is discrainatar'loss when I run lofgan in my own datasets. 
Does this mean the generator not be better after about 7506 epochs?

**Comment by edward3862** [comment]
Using the loss values directly to determine the end of training is usually not recommended for GANs. You can use FID score instead.

**Comment by day2nightya** [comment]
thanks!  I also have another question, that's why you design the loss function of Ladv(D) like this picture? Compared with the loss function ( Ld=-D(X)+D(X') ), what's the advantange of the loss function you designed?
![image](https://user-images.githubusercontent.com/105911615/175288812-5249513f-c487-4035-a2a3-c03cecdb3978.png)

**Comment by edward3862** [comment]
It's the hinge version of GAN loss, which is also widely used in other papers.
